scrapers/lobbyist/scrape_employers.py

```diff
@@ -22,7 +22,7 @@ def _employers(self):
             "pageNumber": 1,
             "pageSize": 1000,
             "sortDir": "ASC",
-            "sortedBy": "",
+            "sortedBy": "Name",
```

Comment:
Add a sort, so that our strategy of using a deque to track seen entity IDs to eliminate dupes actually works. 😅